### PR TITLE
Remove fingerprint for better perf

### PR DESCRIPTION
**Pull request by roberttod**
- The fingerprint is great but it is slow, especially because it checks cookie.
- Use random int twice and add an incrementor to the timestamp in order to spice it up a little bit

### CODE REVIEW

**Comment by roberttod** [comment]
@alanclarke @jimmed (maybe also this function could be moved into slapdash)

**Comment by alanclarke** [comment]
readme is a lie

File: really-unique-id.js [hunk 1]
```diff
@@ -33,20 +36,7 @@ function hammerTime () {
...
- * produced by concatinating strings that are usually
- * unique to each user.
- *
- * @returns {Number} fingerPrintId
- */
-function fingerPrintId () {
-  var maxVal = Math.pow(2, 32) - 1
-  var fingerPrint = navigator.userAgent + document.cookie
-  return format(hash(fingerPrint), maxVal)
+  return format((new Date()).getTime() + incrementor++, doomsDay.getTime())
```
Comment (alanclarke) [inline really-unique-id.js:39]: not sure if doomsday adds any uniqueness? Like its a fixed number of ms from now, but new Date().getTime() is also a fixed number of ms from now? So they're like the same number, but with a slightly different offset

**Comment by alanclarke** [comment]
:+1:

**Pull request merged**